hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JobRecord.java

```diff
@@ -108,6 +108,7 @@ public void readData(ObjectDataInput in) throws IOException {
     public String toString() {
         return "JobRecord{" +
                 "jobId=" + idToString(jobId) +
+                ", jobConfig.name=" + getConfig().getName() +
```

Comment:
we can just say `name=...`

Reply:
This is more technically accurate and matches with `JobResult.toString()`...